config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/AnnotatedBundle.java

```diff
@@ -43,8 +44,7 @@ public String getBundleName() {
     }
 
     public String getUniquePrefix() {
-        AnnotableEntity annotableEntity = (AnnotableEntity) annotatedEntity.getEntity();
-        return projectInfo.getName() + "-" + annotableEntity.getShortenedType() + "-" + PathUtils.extractName(annotatedEntity.getEntityName()) + "-";
+        return projectInfo.getName() + uniqueNameSeparator + PathUtils.extractName(annotatedEntity.getEntityName()) + uniqueNameSeparator;
```

Comment:
entity name is not unique across different entity types, policy and encass can have same name

Reply:
it will have conflicts if encass bundle and policy bundle has same dependent entity

Reply:
the name need not be unique across all entity types.  We will always resolve the entities within its family boundary.

Reply:
Our belief is that project + bundle is globally unique. If it is not the case, let the collisions happen.

Reply:
OK